src/pages/apps/[id]/settings/_components/FormAppSettings.tsx

```diff
@@ -78,13 +112,13 @@ const FormAppSettings = ({
           name="runtime"
           displayEmpty
           value={runtime}
-          onChange={(v) => setRuntime(v)}
+          onChange={e => setRuntime(e.target.value as Runtime)}
```

Comment:
This was the bug. The `Form.Select` API recently changed and this was left out. Instead of providing the value, it now provides the `event` object, so we'll need to access the value by using `e.target.value`.